# 도착 인정 범위 (meters)
goal_range: 2.3

# 장애물 탐지 (거리) 범위 (meters)
ob_dist_range : 3.0 #2.0

# 장애물 탐지 (각도) 범위 (deg)
ob_angle_range : [-100,100] #-100,100

# LiDAR 원본 데이터 보이기/숨기기
show_raw_pcd : 1

# 장애물 좌우에 추가할 안전 각도 (deg)
span_angle : 12 # 12

# 선수각 및 서보모터 이동평균필터 큐 사이즈
filter_queue_size : 3

etk_use : False # extended kalman filiter; if you have True then you can't use fake_gps.launch
#-------------------------------------


# 서보모터 회전 범위 (deg)
rotate_angle_range : [-30, 30] #-90, 90

# 서보모터 제어값 범위
servo_range : [70, 120]

# 서보모터 제어값 결정 상수
angle_alpha : 1.5

# 쓰러스터 PWM 신호 (선속) (1500 정지 ~ 최대 1900)
thruster_speed : 1550
